camel/embeddings/base.py

```diff
@@ -23,35 +25,34 @@ class BaseEmbedding(ABC, Generic[T]):
     @abstractmethod
     def embed_list(
         self,
-        objs: List[T],
+        objs: list[T],
```

Comment:
just curious about the benefit to switch from `List` to `list`. From my understanding `List` is more popular used to do type annotations, from python 3.9 `list` can also do type annotations but I think it would be better to unify the usage for type annotation from `typing`

Reply:
According to https://peps.python.org/pep-0563/ https://peps.python.org/pep-0585/ https://peps.python.org/pep-0604/, IMO using built in type such as `list` is recommended